testing/acceptance_test.py

```diff
@@ -526,7 +526,7 @@ def test_equivalence_pytest_pytest(self):
         assert pytest.main == py.test.cmdline.main
 
     def test_invoke_with_string(self, capsys):
-        retcode = pytest.main("-h")
+        retcode = pytest.main(["-h"])
```

Comment:
this is a invalid test change, `invoke_with_string` is a clear statement

Reply:
Oh good catch, duh.